### PR TITLE
Add python3-ifcfg keys.

### PR DESCRIPTION
**Pull request by nuclearsandwich**
These are available for the selected platforms via the ROS repositories.

We don't have a web index but you can see the packages in the storage pool here: http://packages.ros.org/ros2/ubuntu/pool/main/p/python-ifcfg/

### CODE REVIEW

File: rosdep/python.yaml [hunk 1]
```diff
@@ -5014,6 +5014,12 @@ python3-gnupg:
   fedora: [python3-gnupg]
   gentoo: [dev-python/python-gnupg]
   ubuntu: [python3-gnupg]
+python3-ifcfg:
+  debian:
+    buster: [python3-ifcfg]
+    stretch: [python3-ifcfg]
+  ubuntu:
```
Comment (cottsay) [inline rosdep/python.yaml]: I made this package available in Fedora and EPEL: https://apps.fedoraproject.org/packages/python-ifcfg

```suggestion
  fedora: [python3-ifcfg]
  rhel: ['python%{python3_pkgversion}-ifcfg']
  ubuntu:
```

**Review by cottsay — APPROVED** [review]

**Review by tfoote — APPROVED** [review]
Fine by me. Though we're going to have to revisit this going forward.

**Comment by hidmic** [comment]
`python3-ifcfg` can't be found in Ubuntu 18.04, am I hitting the wrong repo?

**Comment by dirk-thomas** [comment]
Based on the issue ros2/ros2cli#425 it seems that `ifcfg` needs some additional dependencies. The current Debian package doesn't declare any though. Should it add an dependency on e.g. `net-tools`?

@claireyywang FYI

**Comment by dirk-thomas** [comment]
> Based on the issue ros2/ros2cli#425 it seems that `ifcfg` needs some additional dependencies. The current Debian package doesn't declare any though. Should it add an dependency on e.g. `net-tools`?

@claireyywang Can you please make sure that this is being followed up on. Thanks.

**Comment by claireyywang** [comment]
@dirk-thomas  new version of `python3-ifcfg` with `net-tools` dependency is in ROS and ROS2 repo now.

**Pull request merged**